Automattic/vip-governance-plugin
# Restrict plugin to the post editor

## Description

- This PR is meant to restrict the plugin to only the post editor, when a post/page is edited or created. Anywhere else it will be disabled until we properly add support for that in #80. This will fix #79, and ensure that we don't inadvertently repeat this on other pages.
- This will also remove the number formatter library so the plugin is playground compatible. I've also added in a playground link.

## Steps to Test

1. Set up a restricted rules file
2. Load the twentytwentyone theme
3. Go to the widgets screen and ensure the rules don't load
4. Go and edit a post/page, or create a new one and ensure the rules load
5. Go to the site editor and ensure the rules don't load
